<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nedrysoft::RouteAnalyser::LatencyRibbonGroup</class>
 <widget class="QWidget" name="Nedrysoft::RouteAnalyser::LatencyRibbonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>95</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>170</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LatencyRibbonGroup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="Nedrysoft::RouteAnalyser::LatencyWidget" name="idealWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>19</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>ideal</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="Nedrysoft::RouteAnalyser::LatencyWidget" name="warningWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>19</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>warning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Nedrysoft::Ribbon::RibbonLineEdit" name="warningLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="Nedrysoft::RouteAnalyser::LatencyWidget" name="criticalWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>19</height>
        </size>
       </property>
       <property name="text" stdset="0">
        <string>critical</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Nedrysoft::Ribbon::RibbonLineEdit" name="criticalLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Nedrysoft::Ribbon::RibbonLineEdit</class>
   <extends>QLineEdit</extends>
   <header>Ribbon/RibbonLineEdit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Nedrysoft::RouteAnalyser::LatencyWidget</class>
   <extends>QWidget</extends>
   <header>LatencyWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>warningLineEdit</tabstop>
  <tabstop>criticalLineEdit</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
